jsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ESNext",
+    "allowJs": true,
+    "checkJs": true,
+    "moduleResolution": "node",
+    "baseUrl": "./",
+    "paths": {
+      "@/*": ["src/*"]
+    }
+  },
+  "include": ["src"],
+  "exclude": ["node_modules", "dist"]
```

Comment:
`dist` can be removed, the blits repo itself doesn't have a dist folder. We may want to exclude the `packages` folder (for now), unless it doesn't cause any issues

Reply:
Yep - will update! thanks!